services-camel/src/test/java/edu/unc/lib/dl/services/camel/images/AddDerivativeProcessorTest.java

```diff
@@ -109,41 +109,11 @@ public void init() throws Exception {
 
     @Test
     public void createEnhancementTest() throws Exception {
-
-        when(repoObjLoader.getBinaryObject(any(PID.class))).thenReturn(binary);
-        when(binary.getParent()).thenReturn(parent);
-        when(message.getBody()).thenReturn(result);
+        mvFile = moveDir.newFile(fileName + ".PNG");
```

Comment:
this creates the file, which is why it exists in the assertion at the end of the test :) I'd suggest just making a `File` object in the root of the moveDir rather than calling newFile.